### PR TITLE
roslint some packages

### PR DESCRIPTION
**Pull request by fmessmer**
found this branch in my repo

deals with roslint part of #96 
(at least for the packages where I'm involved)

### CODE REVIEW

File: cob_control_mode_adapter/src/cob_control_mode_adapter_node.cpp [hunk 1]
```diff
@@ -1,4 +1,34 @@
...
+ *   Project name: care-o-bot
+ * \note
+ *   ROS stack name: cob_control
+ * \note
+ *   ROS package name: cob_control_mode_adapter
+ *
+ * \author
+ *   Author: Felix Messmer, email: felix.messmer@ipa.fraunhofer.de
+ *
+ * \date Date of creation: September, 201
```
Comment (mgruhler) [inline cob_control_mode_adapter/src/cob_control_mode_adapter_node.cpp]: wow, this is old ;-)

**Review by mgruhler — APPROVED** [review]
approved, but I'd still like @ipa-fxm to fix the date ;-)

File: cob_control_mode_adapter/src/cob_control_mode_adapter_node.cpp (hunk 1 above)
Comment (fmessmer) [reply cob_control_mode_adapter/src/cob_control_mode_adapter_node.cpp]: I do as commanded

Comment (mgruhler) [reply cob_control_mode_adapter/src/cob_control_mode_adapter_node.cpp]: works like a charm :-)

**Review by ipa-nhg — APPROVED** [review]

**Comment by NadiaHG** [comment]
Date fixed https://github.com/ipa320/cob_control/pull/122/commits/b4a2f9f8feb3221a6d2f9cbb6afa9a29ca840f33 --> can be merged!

**Pull request merged**